sale_lead_time_range/models/product.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sale_delay_range_value = fields.Float(
+        "Customer Lead Time",
+        store=True,
+        inverse="_inverse_sale_delay_range_value",
+        help="Delivery lead time. It's the time, promised to the customer, "
+        "between the confirmation of the sales order and the delivery.",
+    )
+
+    sale_delay_range_type = fields.Selection(
+        [
+            ("days", "Days"),
+            ("weeks", "Weeks"),
+            ("months", "Months"),
+            ("years", "Years"),
+        ],
+        string="Range type",
+        default="days",
+        inverse="_inverse_sale_delay_range_value",
+    )
```

Comment:
should it maybe be required? it is confusing if you can have it empty

Reply:
It is required now, can you confirm @ArnauCForgeFlow ?

Reply:
Yes, it is, but in the view